src/pixelator/pixeldataset/utils.py

```diff
@@ -55,7 +56,11 @@ def update_metrics_anndata(adata: AnnData, inplace: bool = True) -> Optional[Ann
     df = adata.to_df()
 
     # update the var layer (antibody metrics)
-    adata.var["antibody_count"] = df.sum().astype(int)
+    # we ignore the warning here, since we actually want to force and update of the
+    # `adata.var` data frame.
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=ImplicitModificationWarning)
+        adata.var["antibody_count"] = df.sum().astype(int)
```

Comment:
Does it only applies to L63 or also to the next two lines L64-65?

Reply:
Only the lines which are encapsulated by the context manager, i.e. only L63.